### DISCUSSION TITLE
laravel package livewire component

### DISCUSSION THREAD

**Discussion by abdosaeedelhassan** [livewire/livewire#1343] [4. General]
i want to develop laravel package that contain livewire components, i don't know how to load livewire component from the package inslide laravel, when i try to @livewire('component-name') i received that Unable to find component, i need help.
thank you

**Comment by Dimonka2** [comment]
You have to register your component in the package ServiceProvider `boot `method like:
```php
 \Livewire\Livewire::component('flatform.table', \dimonka2\flatform\Livewire\TableComponent::class);
```

**Reply by abdosaeedelhassan** [reply]
thank you
